src/community/oseo/oseo-rest/src/main/java/org/geoserver/opensearch/rest/UnknownFieldsSerializer.java

```diff
@@ -0,0 +1,38 @@
+package org.geoserver.opensearch.rest;
```

Comment:
Lacks standard copyright header

Reply:
Added the same copyright of [EncodingHints](https://github.com/geoserver/geoserver/blob/26273424f084bbee64e0e7e672307b8fa8deb03c/src/community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/builders/EncodingHints.java#L1-L4).